code/game/gamemodes/clock_cult/clock_structures/geis_binding.dm

```diff
@@ -16,15 +17,60 @@
 	debris = list()
 	can_buckle = TRUE
 	buckle_lying = 0
-	buckle_prevents_pull = TRUE
-	var/resisting = FALSE
-	var/can_resist = FALSE
+	var/mob/living/resisting = FALSE
 	var/mob_layer = MOB_LAYER
+	var/obj/item/clockwork/slab/slab
+
+/obj/structure/destructible/clockwork/geis_binding/Initialize()
+	. = ..()
+	START_PROCESSING(SSprocessing, src)
+
+/obj/structure/destructible/clockwork/geis_binding/Destroy()
+	STOP_PROCESSING(SSprocessing, src)
+	slab.busy = null
+	slab.icon_state = initial(slab.icon_state)
+	return ..()
+
+/obj/structure/destructible/clockwork/geis_binding/proc/assign_slab(obj/item/clockwork/slab/the_slab) //feed me energy
```

Comment:
this should be in Initialize, with the sleep()ed part under an addtimer() with a time of 1